com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Form/DataSource.php

```diff
@@ -60,8 +58,12 @@ public function preProcess() {
   public function buildQuickForm() {
     //Setting Upload File Size
     $config = CRM_Core_Config::singleton();
-
-    $uploadFileSize = CRM_Core_Config_Defaults::formatUnitSize($config->maxFileSize.'m');
+    if ($config->maxImportFileSize >= 8388608) {
+      $uploadFileSize = 8388608;
+    }
+    else {
+      $uploadFileSize = $config->maxImportFileSize;
+    }
```

Comment:
I don't understand this change. Where did this 8388608 come from? Why we override the value in $config->maxImportFileSize?
Also, the commit says "PR Review Suggestions" (which is pretty generic and will not be very helpful in the future if someone is looking at the commit history) but this was not a suggestion during the PR review